connector_prestashop/views/sale_view.xml

```diff
@@ -0,0 +1,155 @@
+        </field>
+    </record>
+
+    <record id="view_prestashop_sale_order_state_tree" model="ir.ui.view">
+        <field name="name">prestashop.sale.order.state.tree</field>
+        <field name="model">prestashop.sale.order.state</field>
+        <field name="arch" type="xml">
+            <tree string="Sale order State">
+                <field name="backend_id"/>
+                <field name="prestashop_id"/>
+            </tree>
+        </field>
+    </record>
+
+   <record id="view_stock_form" model="ir.ui.view">
+        <field name="model">stock.warehouse</field>
+        <field name="inherit_id" ref="stock.view_warehouse"/>
+        <field name="arch" type="xml">
+            <xpath expr='//notebook' position='before'>
+                <group string="PrestaShop">
+                    <field name="prestashop_bind_ids" nolabel="1">
+                        <tree name="prestashop_binding">
+                            <field name="odoo_id" />
+                            <field name="shop_group_id"/>
+                        </tree>
+                    </field>
+                </group>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="view_prestashop_sale_order_state_form" model="ir.ui.view">
+        <field name="name">prestashop.sale.order.state.form</field>
+        <field name="model">prestashop.sale.order.state</field>
+        <field name="arch" type="xml">
+            <form string="Sale order State">
+                <group>
+                    <field name="backend_id"/>
+                    <field name="prestashop_id"/>
+                </group>
+                <group>
+                    <field name="openerp_state_ids" nolabel="1" mode="tree">
+                        <tree string="States" editable="top">
+                            <field name="name"/>
+                        </tree>
+                    </field>
+                </group>
+            </form>
+        </field>
+    </record>
+
+    <record id="action_prestashop_sale_order_state_form" model="ir.actions.act_window">
+        <field name="name">Sale Order States</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">sale.order.state</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+        <field name="view_id" ref="view_sale_order_state_tree"/>
+        <field name="help" type="html">
+          <p class="oe_view_nocontent_create">
+            Here are the sale order states mapped with e-commerce.
+          </p><p>
+            Each state can be linked between PrestaShop and Odoo.
+            When a sale order state change, the connector will update the sale order in PrestaShop.
+          </p>
+        </field>
+    </record>
+
+    <menuitem
+        id="sale_order_state_tree_menu"
+        name="Sale Order States"
+        action="action_prestashop_sale_order_state_form"
+        parent="sales_team.menu_sale_config"
```

Comment:
Maybe the parent could be sale.menu_sale_config? the sales_team menu don't have a parent, so it's render on the top, and seems ugly have this menu just for the states.